setup.py

```diff
@@ -10,7 +10,7 @@
     author="Department for Levelling Up, Housing and Communities",
     description="Provides access to functionality common to creating a data dashboard.",
     name="gov_uk_dashboards",
-    version="9.13.4",
+    version="9.13.5",
```

Comment:
may need to check package version number as some package PR's were merged yesterday.